src/core/Akka/Actor/ActorPath.cs

```diff
@@ -327,74 +327,165 @@ public static bool TryParse(string path, out ActorPath actorPath)
         {
             actorPath = null;
 
+            if (!TryParseAddress(path, out var address, out var absoluteUri)) return false;
```

Comment:
NIT: would this technically be a relative URI?

Reply:
You can have a relative `ActorSelection` but not a relative `ActorPath` - we never allowed parsing relative ones before either:

https://github.com/akkadotnet/akka.net/blob/c663bc230946644b9d91999d9dbbccf8ebc699f5/src/core/Akka/Actor/ActorPath.cs#L326-L342

This is because in order for the path to compute things like its own string, it has to be able to work its way up the food chain.